docs/content/2.features/5.seo.md

```diff
@@ -0,0 +1,65 @@
+# 4. SEO 
+
+## General info 
+
+SEO is an abbreviate for "Search Engine Optimization".
+Exactpro Docs Toolkit has functionality of adding description and keywords for Google search console.
+
+## Configuration
+
+Add `seo` section to `app.config.ts` file of your docs website, using following template: 
+
+```ts
+export default defineAppConfig({
+  exactproDocs: {
+    seo: {
+      description:
+        '1-2 sentences to briefly describe content of your website',
+      keywords: ['keyword1', 'keyword2', 'keyword3'],
+      robots: [{ UserAgent: '*' }, { Allow: '/' }],
+      sitemap: {
+        baseUrl: 'base URL of your website'
+      },
+      verificationMetaTags: [
+        // <meta name="google-search-console-token" />
+        {
+          name: 'google-site-verification',
+          content: 'google-search-console-token'
+        }
+      ]
+    }
+  }
+})
+```
+
+::notice{note}
+
+You can use `app.config.ts` of our documentation as another example of SEO configuration!
+
+::
+
+## Fields description
+
+### description:
+
+Provide 1-2 sentences to briefly describe content of your website.
+They will be displayed under the link to your website in search results.
+
+### keywords: 
+
+Here you should pass all keywords that describes your website as an array, they will be used to characterise your website for search engine.
+
+### robots: 
+
+You should leave contents of this fields as it is, because it allows all integrated automation to work.
+
+
+### sitemap: { baseUrl: }
+
+You should add an external link to main page of your website.
+It will be used in sitemap building.
+
+### verificationMetaTags:
+
+Leave value of `name` field as it is. 
+Specify Google search console token to the `content` field to prove, that website belongs to you.
```

Comment:
I clearly remember there is useful link about authentication in JSDoc. 

And actually `name` should be provided for specific Search Engine.

Reply:
Used it, thanks